web-dev-server.config.js

```diff
@@ -0,0 +1,25 @@
+import { readFileSync } from 'fs';
+import replace from './rollup/rollup-plugin-replace-simple.js';
+
+const { version } = JSON.parse(readFileSync('./package.json'));
+const buildDate = new Intl.DateTimeFormat().format(new Date());
+
+const webDevReplace = config => {
+	const replaceInstance = replace(config);
+	return {
+		name: 'replace',
+		transform: context => replaceInstance.transform(context.body, context.path)?.code ?? context.body,
+	};
+};
+
+export default {
```

Comment:
If we are going to introduce a `web-dev-server` config file, we could move the ` --node-resolve --watch --open` flags from `package.json` down here.

Reply:
Good call, looks like I'll go with no `web-dev-server` config though.